lms/static/js/learner_dashboard/views/program_details_view.js

```diff
@@ -29,6 +44,16 @@
                     this.headerView = new HeaderView({
                         model: this.programModel
                     });
+                    new CollectionListView({
+                        el: '.js-course-list',
+                        childView: CourseCardView,
+                        collection: this.courseCardsCollection,
+                        context: this.programModel.toJSON(),
+                        titleContext: {
+                            el: 'h2',
+                            title: 'Course List'
```

Comment:
No, this is not the page title. This is the screen reader only title for the whole collection. It's not visible on the page.